identity-enabler/verifier-mobile-app/src/lib/ui/helpers.ts

```diff
@@ -40,3 +41,71 @@ export function getTimeString(date: Date): string {
         minute: "2-digit"
     });
 }
+
+export const credentialDisplayMap = new Map<CredentialType, string>([
```

Comment:
I find it more natural to use just a regular Javascript object to represent hashes but no a big deal
```
export const credentialDisplayMap : { [key: CredentialType]: string } = {
   CredentialType.PERSONAL_INFO: "Personal Information",
 
};
```